_posts/2019/2019-11-03-write-one-test.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Write One Test"
+description: "Much, much better than none"
+layout: article
+category: ruby
+image:
+  base: '2019/write-one-test'
+  alt: 'Smoke'
+  credit: 'Pascal Meier'
+  source: 'https://unsplash.com/photos/1uVCTVSn-2o'
+
+---
+
+Rails comes with a built-in testing framework and many Rubyists eulogise various methods of testing. Behaviour Driven Development, 100% test coverage, Red-green-refactor.
+
+But perhaps you’ve never really ‘got it’ or ever seen the benefits of an established testing culture. As a result you might not have got _any_ tests on your Rails application.
```

Comment:
*...you might not have _any_ tests _in_ your Rails application.

Reply:
```suggestion
But perhaps you’ve never really ‘got it’ or ever seen the benefits of an established testing culture. As a result you might not have _any_ tests in your Rails application.
```